repos/spack_repo/builtin/packages/py_kornia_rs/package.py

```diff
@@ -43,6 +43,9 @@ class PyKorniaRs(PythonPackage):
     # dlpack-rs needs libclang
     depends_on("llvm+clang")
 
+    # Pin fast_image_resize to 5.1.0, >=5.2 requires rust >=1.87
+    patch("py-kornia-rs-pin-fast_image_resize.patch", when="@0.1.9")
```

Comment:
Is this patch also needed to build older versions? Can you add a link to the PR where it was patched in future versions?

Reply:
It was fixed here, seems to be related to a similar issue.

https://github.com/kornia/kornia-rs/commit/93f768137814709b60767f7fc24a9b0184002aee

As far as older versions included in the spack package, they seemed to rely on older versions/different libraries.